components/console/helpers/table.rst

```diff
@@ -181,11 +181,27 @@ The table style can be changed to any built-in styles via
     // same as calling nothing
     $table->setStyle('default');
 
-    // changes the default style to compact
+    // changes the default style to markdown
+    $table->setStyle('markdown');
+    $table->render();
+
+This outputs the table in the Markdown format:
```

Comment:
```suggestion
The output of this command will be:
```

Reply:
I wanted to put some emphasis on that it supports Markdown format ... what do you think?

Reply:
Yes. I think it's not a bad idea to emphasize this, so I merged it "as is". Thanks.